xacro/leg.xacro

```diff
@@ -11,43 +11,92 @@
   <xacro:property name="HFE_2_KFE_z" value="${160.00 * 0.001}" />
 
   <!-- To check this data to get the center of the foot properly -->
```

Comment:
If the data has been checked you should remove this comment as well. Thanks for checking this BTW!

Reply:
Done.